build.gradle

```diff
@@ -35,12 +35,61 @@ springBoot {
 	}
 }
 
+configurations {
+	jaxb
+}
+
+task genJaxb {
+	ext.sourcesDir = "${buildDir}/generated-sources/jaxb"
+	ext.classesDir = "${buildDir}/classes/jaxb"
+	ext.schema = "${projectDir}/src/main/resources/xsd"
+
+	outputs.dir classesDir
+
+	doLast() {
+		project.ant {
+			taskdef name: 'xjc', classname: 'com.sun.tools.xjc.XJCTask',
+					classpath: configurations.jaxb.asPath
+			mkdir(dir: sourcesDir)
+			mkdir(dir: classesDir)
+
+			xjc(destdir: sourcesDir) {
+				produces(dir: sourcesDir, includes: '**/*.java')
+				schema(dir: schema, includes: '**/*.xsd')
+			}
+
+			javac(destdir: classesDir, source: 1.8, target: 1.8, debug: true,
+					debugLevel: 'lines,vars,source',
+					classpath: configurations.jaxb.asPath) {
+				src(path: sourcesDir)
+				include(name: "**/*.java")
+				include(name: "*.java")
+			}
+
+			copy(todir: classesDir) {
+				fileset(dir: sourcesDir, erroronmissingdir: false) {
+					exclude(name: "**/*.java")
+				}
+			}
+		}
+	}
+}
 dependencies {
 	compile('uk.gov.ons.census.fwmt:census-fwmt-canonical:1.0.11-RELEASE')
 	compile('uk.gov.ons.census.fwmt:census-fwmt-events:1.0.13-RELEASE')
-	compile('uk.gov.ons.census.fwmt:census-fwmt-common:1.0.25-RELEASE')
+	compile('uk.gov.ons.census.fwmt:census-fwmt-common:1.0.29-RELEASE')
 	compile('uk.gov.ons.ctp.integration.common:product-reference:0.0.12')
 
+	jaxb("javax.xml.bind:jaxb-api:2.3.0")
```

Comment:
Why is xml binding needed, I cant see anything in the code that needs the biinding stuff

Reply:
Not quite sure on this as I can't find it within the build.gradle

Reply:
Removed JAXB